guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -144,12 +144,27 @@ endif::[]
 ----
 # leapp preupgrade
 ----
-The first run is expected to fail but report issues.
```

Comment:
I think I would keep that here and reword it to something like:
"The first run is expected to report issues and inhibit the upgrade."
And then we can go on and list the possible issues.

Technically, I would probably also merge the next step ("Examine the report") into this one, but probably not in this PR.